content/contact.md

```diff
@@ -6,7 +6,11 @@ weight: 5
 
 # Contact
 
-If you have any questions or want to the contact the BAPC 2021 committee, please drop us an email at: organization “at” BAPC “dot” EU.
+The BAPC 2021 is hosted by [Vrije Universiteit Amsterdam](https://www.vu.nl/), and organized by a committee of [study association STORM](https://storm.vu). If you have any questions or want to contact the BAPC 2021 committee, these are the ways to do it:
+
+***Mail*** — You can drop us an email at: *organization “at” BAPC “dot” EU*.
```

Comment:
Move this one down?

And than tell people this is the less preferred alternative?